lighthouse-cli/test/smokehouse/test-definitions/screenshot/expectations.js

```diff
@@ -12,9 +12,39 @@ const expectations = [
   {
     artifacts: {
       FullPageScreenshot: {
-        width: '>1000',
-        height: '>1000',
-        data: /data:image\/jpeg;base64,.{10000,}$/,
+        screenshot: {
+          width: '>1000',
+          height: '>1000',
+          data: /data:image\/jpeg;base64,.{10000,}$/,
+        },
+        nodes: {
+          'page-0-BODY': {
+            top: 8,
+            bottom: 1008,
+            left: 8,
+            right: 1008,
+            width: 1000,
+            height: 1000,
+          },
+          'page-1-P': {
```

Comment:
is this the one that's exercising the movement test? seems like it's at the very top already and wouldn't have moved...

a comment on the one that's specifically trying to test for update after movement would be nice :)